moodle-cluster-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,665 @@
+          {
+            "name":"SSH-VM0",
+            "properties": {
+              "frontendIPConfiguration": {
+                "id":"[variables('frontEndIPConfigID')]"
+              },
+              "protocol": "tcp",
+              "frontendPort": 2200,
+              "backendPort": 22,
+              "enableFloatingIP": false
+            }
+          },
+          {
+            "name":"Web-VM1",
+            "properties": {
+              "frontendIPConfiguration": {
+                "id":"[variables('frontEndIPConfigID')]"
+              },
+              "protocol": "tcp",
+              "frontendPort": 8081,
+              "backendPort": 8080,
+              "enableFloatingIP": false
+            }
+          },
+          {
+            "name":"SSH-VM1",
+            "properties": {
+              "frontendIPConfiguration": {
+                "id":"[variables('frontEndIPConfigID')]"
+              },
+              "protocol": "tcp",
+              "frontendPort": 2201,
+              "backendPort": 22,
+              "enableFloatingIP": false
+            }
+          },
+          {
+            "name":"Web-VM2",
+            "properties": {
+              "frontendIPConfiguration": {
+                "id":"[variables('frontEndIPConfigID')]"
+              },
+              "protocol": "tcp",
+              "frontendPort": 8082,
+              "backendPort": 8080,
+              "enableFloatingIP": false
+            }
+          },
+          {
+            "name":"SSH-VM2",
+            "properties": {
+              "frontendIPConfiguration": {
+                "id":"[variables('frontEndIPConfigID')]"
+              },
+              "protocol": "tcp",
+              "frontendPort": 2202,
+              "backendPort": 22,
+              "enableFloatingIP": false
+            }
+          },
+          {
+            "name":"Web-VM3",
+            "properties": {
+              "frontendIPConfiguration": {
+                "id":"[variables('frontEndIPConfigID')]"
+              },
+              "protocol": "tcp",
+              "frontendPort": 8083,
+              "backendPort": 8080,
+              "enableFloatingIP": false
+            }
+          },
+          {
+            "name":"SSH-VM3",
+            "properties": {
+              "frontendIPConfiguration": {
+                "id":"[variables('frontEndIPConfigID')]"
+              },
+              "protocol": "tcp",
+              "frontendPort": 2203,
+              "backendPort": 22,
+              "enableFloatingIP": false
+            }
+          },
+          {
+            "name":"Web-VM4",
+            "properties": {
+              "frontendIPConfiguration": {
+                "id":"[variables('frontEndIPConfigID')]"
+              },
+              "protocol": "tcp",
+              "frontendPort": 8084,
+              "backendPort": 8080,
+              "enableFloatingIP": false
+            }
+          },
+          {
+            "name":"SSH-VM4",
```

Comment:
It is one NAT rule per front end VM (with an upper limit of 5). We have explained the mapping in the readme:  https://github.com/vinhub/azure-quickstart-templates/tree/moodle-opensis-pr/moodle-cluster-ubuntu#front-end-vms